app/models/eligibility_verification.rb

```diff
@@ -2,11 +2,24 @@
 
 class EligibilityVerification < ApplicationRecord
   belongs_to :passenger
+  # belongs_to set to optional true with presence validation in order to
+  # make bootstrap validations work nicely.
+  belongs_to :verifying_agency, optional: true
```

Comment:
You might not want  `optional: true`. This allows for a `verifying_agency_id` that doesn't go with any actual `verifying_agency`.

Reply:
Although, on second thought, it really is optional.  Two opinions — that are a little at odds — I guess:

1.  The users are never presented with the opportunity to, I guess, type in an agency id, so it will always be one in the list. Then the only concern is what happens if an agency is deleted. Perhaps a `dependent: restrict` is warranted?
2.  Maybe it's a good idea to also add `validates :verifying_agency, etc`?

Reply:
I did validate the verifying agency id under certain conditions. I think the only concern I have is someone deleting it, so I'll add the dependent: restrict.
If some yahoo gets hacky and adds a verifying agency ID that doesn't exist I won't lose sleep over it.